docs/api/render.md

```diff
@@ -1,6 +1,7 @@
 # Static Rendering API
 
-enzyme's `render` function is used to render react components to static HTML and analyze the
+Use enzyme's `render` function to generate HTML text from your React tree, 
```

Comment:
"HTML text" is a contradiction; something is either HTML, or text. "static HTML" is the proper term here.

Reply:
and I thought of 'static' as meaning, 'class-based' or 'standing in one spot', neither of which made sense.  how about just 'HTML'?